L/libint/build_tarballs.jl

```diff
@@ -0,0 +1,60 @@
+# Note that this script can accept some limited command-line arguments, run
+# `julia build_tarballs.jl --help` to see a usage message.
+using BinaryBuilder, Pkg
+
+name = "libint"
+version = v"2.7.0-beta.5"
+
+# Collection of sources required to complete build
+sources = [
+    ArchiveSource("https://github.com/evaleev/libint/releases/download/v2.7.0-beta.5/libint-2.7.0-beta.5.tgz", "38c630d9b4433340079ebc7a93f05d7a285833883327bfc35dd208249e06f05f"),
+    GitSource("https://gitlab.com/libeigen/eigen.git", "21ae2afd4edaa1b69782c67a54182d34efe43f9c"),
+    DirectorySource("./bundled")
+]
+
+# Bash recipe for building across all platforms
+script = raw"""
+cd $WORKSPACE/srcdir
+for f in ${WORKSPACE}/srcdir/patches/*.patch; do
+    atomic_patch -p1 ${f}
+done
+cd eigen/
+mkdir build
+cd build/
+cmake -DCMAKE_INSTALL_PREFIX=$prefix -DCMAKE_TOOLCHAIN_FILE=${CMAKE_TARGET_TOOLCHAIN} ..
+apk add boost-dev
+make install
+cd ../../
+cd libint-2.7.0-beta.5/
+mkdir build
+cd build/
+cmake -DCMAKE_INSTALL_PREFIX=$prefix -DCMAKE_TOOLCHAIN_FILE=${CMAKE_TARGET_TOOLCHAIN} -DCMAKE_BUILD_TYPE=Release ..
+cmake --build . --target install -- -j${nprocs}
+"""
+
+# These are the platforms we will build for by default, unless further
+# platforms are passed in on the command line
+platforms = [
+    Linux(:i686, libc=:glibc),
+    Linux(:x86_64, libc=:glibc),
+    Linux(:aarch64, libc=:glibc),
+    Linux(:armv7l, libc=:glibc, call_abi=:eabihf),
+    Linux(:powerpc64le, libc=:glibc),
+    Linux(:i686, libc=:musl),
```

Comment:
Curious that `x86_64-linux-musl` is missing here :thinking:   Did the build fail for you locally?

Reply:
yes it did, I'm not sure why. The build was not verbose and I wasn't sure how to troubleshoot it.

Reply:
Is fixing this necessary for merge?

Reply:
Not really necessary, but I'm very surprised that this platform in particular would fail, especially if all other Musl platforms work

Reply:
Yeah frankly I've been working on this build for too long already so I just disabled the platform to get it done ...